src/wizard/wizard-form.tsx

```diff
@@ -88,7 +103,9 @@ export default function WizardForm({
         errorText={errorText}
         errorIconAriaLabel={i18nStrings.errorIconAriaLabel}
       >
-        {content}
+        <div role="region" aria-label={title}>
```

Comment:
The wizard test-util wrapper inherits from the form wrapper which has method `findContent`. We might want to override the method to ensure it returns the client `content` ignoring the new region wrapper. A dry-run to live would also make sense.